mmv1/third_party/terraform/resources/resource_tags_location_tag_bindings.go.erb

```diff
@@ -0,0 +1,351 @@
+		for pageToken != "" {
+			url, err = addQueryParams(url, map[string]string{"pageToken": fmt.Sprintf("%s", res["nextPageToken"])})
+			if err != nil {
+				return handleNotFoundError(err, d, fmt.Sprintf("TagsLocationTagBinding %q", d.Id()))
+			}
+			res, err = sendRequest(config, "GET", billingProject, url, userAgent, nil)
+			if err != nil {
+				return handleNotFoundError(err, d, fmt.Sprintf("TagsLocationTagBinding %q", d.Id()))
+			}
+			if res == nil {
+				// Object isn't there any more - remove it from the state.
+				log.Printf("[DEBUG] Removing TagsLocationTagBinding because it couldn't be matched.")
+				d.SetId("")
+				return nil
+			}
+
+			v, ok := res["tagBindings"]
+			if !ok || v == nil {
+				return nil
+			}
+			pView = append(pView, v.([]interface{})...)
+
+			if token, ok := res["nextPageToken"]; ok {
+				pageToken = token.(string)
+			} else {
+				pageToken = ""
+			}
+		}
+	}
+
+	newMap := make(map[string]interface{}, 1)
+	newMap["tagBindings"] = pView
+	res, err = flattenNestedTagsLocationTagBinding(d, meta, newMap)
+	if err != nil {
+		return err
+	}
+
+	if err := d.Set("name", flattenNestedTagsLocationTagBindingName(res["name"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+	if err := d.Set("parent", flattenNestedTagsLocationTagBindingParent(res["parent"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+	if err := d.Set("tag_value", flattenNestedTagsLocationTagBindingTagValue(res["tagValue"], d, config)); err != nil {
+		return fmt.Errorf("Error reading LocationTagBinding: %s", err)
+	}
+
+	return nil
+}
+
+func resourceTagsLocationTagBindingDelete(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	userAgent, err := generateUserAgentString(d, config.userAgent)
+	if err != nil {
+		return err
+	}
+
+	billingProject := ""
+
+	url, err := replaceVars(d, config, "{{TagsLocationBasePath}}{{name}}")
+	if err != nil {
+		return err
+	}
+
+	var obj map[string]interface{}
+	log.Printf("[DEBUG] Deleting LocationTagBinding %q", d.Id())
+
+	// err == nil indicates that the billing_project value was found
+	if bp, err := getBillingProject(d, config); err == nil {
+		billingProject = bp
+	}
+
+	res, err := sendRequestWithTimeout(config, "DELETE", billingProject, url, userAgent, obj, d.Timeout(schema.TimeoutDelete))
+	if err != nil {
+		return handleNotFoundError(err, d, "LocationTagBinding")
+	}
+
+	err = tagsOperationWaitTime(
+		config, res, "Deleting LocationTagBinding", userAgent,
+		d.Timeout(schema.TimeoutDelete))
+
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[DEBUG] Finished deleting LocationTagBinding %q: %#v", d.Id(), res)
+	return nil
+}
+
+func resourceTagsLocationTagBindingImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	config := meta.(*Config)
+	
+	// current import_formats can't import fields with forward slashes in their value
```

Comment:
Yeah @slevenick, I had tried different combination for this import but not getting expected results and also want to mention the same type of resource example : (https://github.com/hashicorp/terraform-provider-google/blob/213ff96446b18bffa4060e2f434c9f3f47599c78/google/resource_tags_tag_binding.go#L252), in that also import is not working because of multiple forward slashes in import field value . So could you suggest any other better option for that or should I need to remove support for import ?

Reply:
The code linked in the resource_tags_tag_binding should be pretty similar as to what is needed here.

I think the main issue is a lack of a correct ID format for this resource. It should probably be only `name`

Reply:
So in this case of `resource_tags_location_tag_binding` I have tried to set ID as `name` but the problem with that is it doesn't resolve the host for API call as it's baseURL includes the `{{location}}` variable in it. Apart from that I have tried to include `location` in regex but as this attribute is not involved in any other operation except in `base URL` , the import is not getting done successfully.

Reply:
Adding the location into the ID and parsing it during import should be an option. Does that not work correctly?

You may need to change the ID of the resource to support this

Reply:
Thanks for the help @slevenick, Import is working now.